autoload/test/php/phpspec.vim

```diff
@@ -3,7 +3,9 @@ if !exists('g:test#php#phpspec#file_pattern')
 endif
 
 function! test#php#phpspec#test_file(file) abort
-  return a:file =~# g:test#php#phpspec#file_pattern
+  if a:file =~# g:test#php#phpspec#file_pattern
+    return search("PhpSpec") > 0
```

Comment:
I wanted to search for `PhpSpec\ObjectBehavior` but it doesn't seem to like slashes in a search query and I couldn't figure out how to escape the,